docs/validator/economics.md

```diff
@@ -0,0 +1,134 @@
+However, each shard allows a fixed number of *seats*, with validators randomly assigned to one (or more) of them between every `epoch`. The more the shards, the higher the number of available *seats* - so, the more the *seats*, the larger the room for new validators.
+
+As a validator, you need to secure three classes of resources:
+1. NEAR tokens to stake
+2. Nodes infrastructure
+3. Continuous operation
+
+
+## 1. NEAR tokens to stake
+
+NEAR protocol determines validators at the end of each `epoch`, ordering them by the size of their stake. Current validators are re-elected automatically, by *re-staking* their tokens plus the accrued rewards; new validators may replace an existing one if they offer a more significant stake.
+
+As a validator, you can obtain a *seat* by:
+- Staking your NEAR tokens
+- Staking delegated NEAR tokens
+
+<blockquote class="info">
+    <strong>did you know?</strong><br><br>
+    A large enough stake can obtain more than one seat, multiplying the rewards. 
+</blockquote>
+
+### Stake your NEAR tokens
+
+Use [NEAR Shell](../development/near-clitool.md) to issue a staking transaction and append your transaction to the `proposals`. At the end of the `epoch`, if your proposal is sufficient, your node will become a validator - and will begin to generate chunks/blocks after one more epoch. You will be able to monitor the `proposals`, and eventually top-up your transaction to increase your odds.
+
+### Stake delegated NEAR tokens
+
+NEAR leverages *smart contracts* for delegation. Therefore, every validator should implement its staking contract, or use one already available among the community. 
+This option is still work in progress, so please follow the [NEP on GitHub](https://github.com/nearprotocol/NEPs/blob/staking-contract/text/0000-staking-contract.md) for the specifications, and add your comments in the 'Riskless Delegation' discussion on [NEAR's Commonwealth](https://commonwealth.im/near/proposal/discussion/357-riskless-delegation-aka-tezos-delegation).
+
+<blockquote class="warning">
+    <strong>heads up</strong><br><br>
+    In the beginning, the network will run on a limited number of shards, with all validators competing for a limited number of seats.
+    As a simple rule-of-thumb, distributing 40% of the total supply (400 million out of 1 billion NEAR tokens) between 100 seats will need an average of 4 million NEAR tokens per validator.
+</blockquote>
+
+
+## 2. Nodes infrastructure
+
+Once you obtained your NEAR stake, you need an infrastructure to serve the network securely.
+At the beginning of a new `epoch`, the protocol randomly assigns every validator to one of these roles:
+- Chunk Producer, for collecting transactions and applying them to the state
+- Block Producers, for generating the single blocks containing the current chunks 
+- Fishermen (aka 'hidden' validators), for challenging the state to verify the work of other validators
```

Comment:
This is not true. Chunk producers are always block producers as well. Fishermen are rather separate. They are people who stake voluntarily for the safety of the network. They do not gain any reward through staking.

Reply:
Awesome, so can I simply call them block producers and avoid any confusion around the term "chunks". Do you think that leaving a small footnote to clarify that will work? It may link to the part of the Nightshade Paper that instead separates these two.

Regarding fishermen: are they anyway submitting their proposal, or they simply run a node in a different state, by issuing a particular staking transaction?

Reply:
> Do you think that leaving a small footnote to clarify that will work?

I think that's fine.


> Regarding fishermen: are they anyway submitting their proposal, or they simply run a node in a different state, by issuing a particular staking transaction?

They also submit proposal but there is a threshold for becoming fishermen which will be much smaller than the threshold for becoming validators. So for people who want to become fishermen, it makes sense for them to stake exactly the threshold amount.